go.mod

```diff
@@ -74,6 +74,8 @@ replace git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110
 
 replace k8s.io/client-go => k8s.io/client-go v0.19.2
 
+replace k8s.io/api => k8s.io/api v0.19.4
```

Comment:
This is not used in any Cortex or vendored packages. It was actually used in tests of vendored package (Prometheus), so this replace should be safe (version was picked from what was present in go.sum).